user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/GQIArgument.md

```diff
@@ -28,3 +30,19 @@ In addition, the following derived types are supported from DataMiner 10.3.0/10.
 - `GQIStringDropdownArgument`
 
 - `GQIStringListArgument`
+
+Arguments only available for operators:
```

Comment:
This could maybe be reworded a bit more positively
```suggestion
Additionally, for custom operators you can also use the following arguments to select existing columns.
```